sources/niLang/src/DataTablePath.h

```diff
@@ -56,10 +55,12 @@ enum eDataTablePathOp
   eDataTablePathOp_Parent = 2,
   //! Path operation, datatable.
   eDataTablePathOp_DataTable = 3,
+  //! Path operation, datatable with predicate.
+  eDataTablePathOp_Predicate = 4,
   //! Path operation, property.
-  eDataTablePathOp_Property = 4,
+  eDataTablePathOp_Property = 5,
```

Comment:
I think I already commented on this, but keep the original order - put Predicate at 5 and leave Property at 4. Prefer preserving the order of enums by default.